app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -242,6 +246,9 @@ private void updateCurrentBgInfoForBtBasedWixel(TextView notificationText) {
     }
 
     private void updateCurrentBgInfoCommon(TextView notificationText) {
+        if (alreadyDisplayedBgInfoCommon) return; // with bluetooth and wifi, skip second time
```

Comment:
Where is `alreadyDisplayedBgInfoCommon` set to `false` again?
At the moment it looks like there would be no updates to the number when `updateCurrentBgInfo()` gets called every minute.

I personally prefer control flow without state variables (like 'alreadyDisplayedBgInfoCommon' ) if I can do something like this:
```java
    private void updateCurrentBgInfoForWifiWixel(TextView notificationText) {
        if (!WixelReader.IsConfigured(getApplicationContext())) {
            notificationText.setText("First configure your wifi wixel reader ip addresses");
            return;
        }
        if (!isWifiBluetoothWixel){ //allready called for BT on BTWifiWixel
            updateCurrentBgInfoCommon(notificationText);
        }
    }
```
But that's just my preference. I believe it easier to debug.